lib/tasks/fetch_latest_db.rake

```diff
@@ -15,7 +15,7 @@ task :fetch_latest_db => :environment do
 
   puts "Restoring the database with #{backup.name}"
   backup_filepath = fetch_file_path(backup)
-  system("pg_restore --clean --no-acl --no-owner -h localhost -d diaper_dev #{backup_filepath}")
+  system("pg_restore --clean --no-acl --no-owner -h localhost -d diaper_dev -U postgres #{backup_filepath}")
```

Comment:
Not sure if this one will break anyone. I don't think so, but would love verification.

Reply:
It might, if someone happens to have set up their local Postgres differently than the default. Can we allow for a `LOCAL_POSTGRES_USER` argument to this task?

Reply:
Sure -- do you think it should be the ENV PG_USERNAME or an actual (optional) rake task param? I'm leaning toward the ENV

Reply:
ENV can be fine.

Reply:
Pretty sure the "default default" is postgres, which should be fine. If it doesn't work, people can always add the env variable themselves.